docarray/helper.py

```diff
@@ -321,3 +322,69 @@ def dataclass_from_dict(klass, dikt):
         if isinstance(dikt, (tuple, list)):
             return [dataclass_from_dict(klass.__args__[0], f) for f in dikt]
         return dikt
+
+
+def protocol_and_compress_from_file_path(
+    file_path: str,
+    default_protocol: Optional[str] = None,
```

Comment:
what if it is not found and this is None, will it fail?

Reply:
No, since `load_binary` have default values.